pipeline/src/main/java/edu/rosehulman/aixprize/pipeline/annotators/ColorsAnnotator.java

```diff
@@ -0,0 +1,99 @@
+package edu.rosehulman.aixprize.pipeline.annotators;
+
+import java.io.*;
+import java.net.*;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.*;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.*;
+import org.apache.uima.UimaContext;
+import org.apache.uima.analysis_component.JCasAnnotator_ImplBase;
+import org.apache.uima.analysis_engine.AnalysisEngineProcessException;
+import org.apache.uima.jcas.JCas;
+import org.apache.uima.jcas.tcas.Annotation;
+import org.apache.uima.json.JsonCasSerializer;
+import org.apache.uima.resource.ResourceInitializationException;
+import org.json.*;
+
+import edu.rosehulman.aixprize.pipeline.types.Color;
+
+/**
+ * Communicates with a Python script to find color words in a file. 
+ * Interface: 
+ * - writes a standard JSON-ified CAS to the Socket at port 8080 
+ * - receives a series of annotations in JSON, one per line, with fields of: 
+ *   + begin: int 
+ *   + end: int 
+ *   + name: String 
+ * - the last annotation line should be an empty JSON object
+ */
+public class ColorsAnnotator extends JCasAnnotator_ImplBase {
+
+	private URI uri;
+	private CloseableHttpClient client;
+
+	@Override
+	public void initialize(UimaContext aContext) throws ResourceInitializationException {
+		try {
+			this.uri = new URIBuilder().setHost("127.0.0.1")
+									   .setScheme("http")
```

Comment:
I think it is only displaying like this because of the browser tab width. I would be open to switching to spaces, but I didn't want 1000 line inserts of whitespace changes to move the whole file over.